plugin/Renderer/InputModelRenderer.php

```diff
@@ -219,6 +219,7 @@ private function createInputField(InputModelBase $pInputModel, FormModel $pFormM
 			case InputModelOption::HTML_TYPE_EMAIL:
 				$pInstance = new InputFieldEmailRenderer('email', $elementName);
 				$pInstance->addAdditionalAttribute('size', '50');
+				$pInstance->addAdditionalAttribute('pattern', '^[\w.-]+@[\w.-]+\.[\w]{2,}$');
```

Comment:
why not just `<input type="email">`?

Reply:
`type="email"` only check @ sign.  For example, 'test@gmail' is considered a valid email in that case. I added `pattern`  attribute to make it cover more invalid cases

Reply:
Yes, email validation is suprisingly difficult.

I'm going to decide that we don't need `pattern` validation and `type="email"` is enough.

This is supported by https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#pattern_validation, where they say you only need pattern validation if you want the inputs to be 'restricted further than just "any string that looks like an e-mail address"'. This is not the case for us, we accept all email addresses here. If the user enters something invalid, we can't really do much about it.

Therefore, please remove the `pattern`. :)

Reply:
ok, I removed it